client/src/main/scala/com/lnvortex/client/VortexClient.scala

```diff
@@ -652,7 +654,7 @@ case class VortexClient[+T <: VortexWalletApi](vortexWallet: T)(implicit
         for {
           _ <- vortexWallet.broadcastTransaction(signedTx)
           _ <- vortexWallet
-            .labelTransaction(signedTx.txId, s"LnVortex Mix Anon set: $anonSet")
+            .labelTransaction(signedTx.txId, s"Vortex Anon set: $anonSet")
```

Comment:
```suggestion
            .labelTransaction(signedTx.txId, s"Vortex Anonimity set: $anonSet")
```

Reply:
Reverted this change to be done in your PR